### PR TITLE
Action config validator 2.0-dev branch

### PR DESCRIPTION
**Pull request by mwwolters**
Thanks for opening a Pull Request!

Here's a handy checklist to ensure your PR goes smoothly.

- [x] I signed Google's [Contributor License Agreement](https://opensource.google.com/docs/cla/)
- [x] My code conforms to Google's [python style](https://google.github.io/styleguide/pyguide.html).
- [x] My PR at a minimum doesn't decrease unit-test coverage (if applicable).
- [x] My PR has been functionally tested.
- [x] All of the [unit-tests](http://forsetisecurity.org/docs/development/#executing-tests) still pass.
- [x] Running `pylint --rcfile=pylintrc` passes.

These guidelines and more can be found in our [contributing guidelines](https://github.com/GoogleCloudPlatform/forseti-security/blob/master/.github/CONTRIBUTING.md).

---
This change is [Reviewable](https://reviewable.io/reviews/googlecloudplatform/forseti-security/857)

### CODE REVIEW

File: google/cloud/forseti/actions/action_config_validator.py [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+        for err in action_errs:
+            if err is not None:
+                errors.append(err)
+
+    if errors:
+        raise ConfigLoadError(errors)
+
+    return config
+
+def _load_and_validate_yaml(path):
```
Comment (ahoying) [inline google/cloud/forseti/actions/action_config_validator.py]: Please add tests.

File: google/cloud/forseti/actions/action_config_validator.py [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+# limitations under the License.
+
+"""Actions configuration validator."""
+
+import importlib
+import os
+import yaml
+
+
+def validate(actions_config_path):
```
Comment (ahoying) [inline google/cloud/forseti/actions/action_config_validator.py]: Please add tests.

**Comment by ahoying** [comment]
Also please add the boilerplate header and a docstring to the __init__.py files.

File: google/cloud/forseti/actions/action_config_validator.py (hunk 2 above)
Comment (mwwolters) [reply google/cloud/forseti/actions/action_config_validator.py]: Done.

File: google/cloud/forseti/actions/action_config_validator.py (hunk 1 above)
Comment (mwwolters) [reply google/cloud/forseti/actions/action_config_validator.py]: Done.

**Pull request merged**